src/listeners/messages/message_changed.py

```diff
@@ -0,0 +1,29 @@
+from logging import getLogger
+
+from slack_bolt import App
+
+from src.config.language import language
+from src.rules.pattern import pattern
+
+logger = getLogger(__name__)
+
+
+async def register(app: App):
+    app.event("message")(message_changed_callback)
+
+
+async def message_changed_callback(event, say):
```

Comment:
What do you think about create some unit test here? I known that we don't have unit test for message actions yet, but if you do this, we can improve the message flow with more testes (I must do that soon)